uniswap-bot/src/uniswapABI.ts

```diff
@@ -0,0 +1,30 @@
+export const swapEvent =
+  'event Swap (address indexed sender, address indexed recipient, int256 amount0, int256 amount1, uint160 sqrtPriceX96, uint128 liquidity, int24 tick)';
+
+export const uniswapV3FactoryAddress = '0x1F98431c8aD98523631AE4a59f267346ea31F984';
+
+export const pool_init_hash_code = '0xe34f199b19b2b4f47f68442619d555527d244f78a3297ea89325f843f87b8b54';
+
+export const uniswapPoolABI = [
+  {
+    inputs: [],
+    name: 'fee',
+    outputs: [{ internalType: 'uint24', name: '', type: 'uint24' }],
+    stateMutability: 'view',
+    type: 'function',
+  },
+  {
+    inputs: [],
+    name: 'token0',
+    outputs: [{ internalType: 'address', name: '', type: 'address' }],
+    stateMutability: 'view',
+    type: 'function',
+  },
+  {
+    inputs: [],
+    name: 'token1',
+    outputs: [{ internalType: 'address', name: '', type: 'address' }],
+    stateMutability: 'view',
+    type: 'function',
+  },
+];
```

Comment:
this is not wrong but can also be written like this:

```
export const uniswapPoolABI = [
  "function token0() external view returns (address)",
  "function token1() external view returns (address)",
  "function fee() external view returns (uint24)",
];
```

Reply:
Yes, I just copied the ABI from the etherscan explorer and removed the unused methods. But this is more clean and readable, I will update it.